### DISCUSSION TITLE
Emissive materials are dim compared to the background colour

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3221] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7875817/155230213-39af2bda-6854-4f64-bdd0-4885bae925ee.png)

I have an issue with this light model's emissive material. The light seems darker then the background colour, where I would think the emissive material should be brighter.

Does anyone have any idea what is happening here and how I can make the emissive material brighter?

**Comment by elalish** [comment, marked as answer]
Yes, there's a glTF extension in the works that will help with this, basically allowing the emissive factor to go above 1. You *might* even be able to manage it with `mv.materials[0].setEmissiveFactor([1000, 1000, 1000]);` though I haven't tested it.

**Reply by ghost** [reply]
I'll give that work around a go thank you!
I have noticed that a lot of other viewers set emission to be full white if it is set to 1, I am curious as to why model-viewer would be different in this regard?

**Reply by ghost** [reply]
Setting the Emissive Factor works!

**Reply by elalish** [reply]
Great! The reason the emissive = 1  doesn't come through as white is due to tone mapping, which is used to make a smooth transition between bright and less bright lights.
